### PR TITLE
Update code packages

### LINKED ISSUE #32: Refactor codebase for PlannerMd
Change occurrences of Addressbook in the codebase to PlannerMd

### PR DESCRIPTION
**Pull request by jiayi1129**
Resolves #32

### CODE REVIEW

File: src/main/java/seedu/plannermd/MainApp.java [hunk 1]
```diff
@@ -167,13 +167,13 @@ protected UserPrefs initPrefs(UserPrefsStorage storage) {
...
     public void start(Stage primaryStage) {
-        logger.info("Starting AddressBook " + MainApp.VERSION);
+        logger.info("Starting PlannerMd " + MainApp.VERSION);
         ui.start(primaryStage);
     }
 
     @Override
     public void stop() {
-        logger.info("============================ [ Stopping Address Book ] =============================");
+        logger.info("============================ [ Stopping PlannerMD ] =============================");
```
Comment (pualixue) [inline src/main/java/seedu/plannermd/MainApp.java:176]: Not really an issue but maybe can make it the same as line 51 (`PlannerMd`) for consistency?

**Review by pualixue — COMMENTED** [review]
LGTM, only a minor nitpick.
Just to double confirm, does the app still run as expected?

File: src/main/java/seedu/plannermd/MainApp.java (hunk 1 above)
Comment (jiayi1129) [reply src/main/java/seedu/plannermd/MainApp.java:176]: For logging I would believe that we want the name to still be `PlannerMD`? I believe for naming wise we would still want the name to be `PlannerMD` but for code we would put it as `plannerMd`.

Comment (jiayi1129) [reply src/main/java/seedu/plannermd/MainApp.java:176]: Also i ran the tests and pass all the checks, as well as ran the app and tested out a few functionality manually. Seems to work fine.

File: src/main/java/seedu/plannermd/MainApp.java [hunk 2]
```diff
@@ -48,16 +48,16 @@
 
     @Override
     public void init() throws Exception {
-        logger.info("=============================[ Initializing AddressBook ]===========================");
+        logger.info("=============================[ Initializing PlannerMd ]===========================");
```
Comment (pualixue) [inline src/main/java/seedu/plannermd/MainApp.java:51]:
```suggestion
        logger.info("=============================[ Initializing PlannerMD ]===========================");
```

File: src/main/java/seedu/plannermd/MainApp.java (hunk 1 above)
Comment (pualixue) [reply src/main/java/seedu/plannermd/MainApp.java:176]: Then perhaps we could apply the changes in the comment below, and we are good to merge this PR.

File: src/main/java/seedu/plannermd/MainApp.java (hunk 2 above)
Comment (jiayi1129) [reply src/main/java/seedu/plannermd/MainApp.java]: Good catch! Have resolved this in the following commit

**Pull request merged**